apps/api/src/app/app.module.ts

```diff
@@ -20,19 +20,14 @@ import { DocumentProviderModule } from '@island.is/api/domains/document-provider
 
 const debug = process.env.NODE_ENV === 'development'
 const playground = debug || process.env.GQL_PLAYGROUND_ENABLED === 'true'
-const autoSchemaFile = debug ? 'apps/api/src/api.graphql' : true
 
-/*
- * When adding new resolvers through your modules don't forget to add them to buildSchema.ts as well.
- * So the automatically generated schemas won't be failing when running.
- */
 @Module({
   controllers: [HealthController],
   imports: [
     GraphQLModule.forRoot({
       debug,
       playground,
-      autoSchemaFile,
+      autoSchemaFile: 'apps/api/src/api.graphql',
```

Comment:
Not sure this will work in the Docker container. The account running the process does not have permissions to write to the file system

Reply:
Yeah fixed it. Now in-memory for production builds.

Reply:
Yuup, fixed it. Uses in-memory for production.